bindings/msvc/samples/main.c

```diff
@@ -22,7 +22,7 @@
 
 // posix specific
 #else // _MSC_VER
-#include "platform.h"
+#include "unicorn/platform.h"
```

Comment:
the line above is not needed, because we include `unicorn/unicorn.h` in the next line, which includes `platform.h` inside